apps/examples/webserver/webserver_main.c

```diff
@@ -441,12 +367,12 @@ pthread_addr_t httptest_cb(void *arg)
 		printf("Error: Cannot allocate server structure!!\n");
 		goto release;
 	}
-	ssl_config.root_ca = (char *)ca_crt_rsa;
-	ssl_config.root_ca_len = sizeof(ca_crt_rsa);
-	ssl_config.dev_cert = (char *)srv_crt_rsa;
-	ssl_config.dev_cert_len = sizeof(srv_crt_rsa);
-	ssl_config.private_key = (char *)srv_key_rsa;
-	ssl_config.private_key_len = sizeof(srv_key_rsa);
+	ssl_config.root_ca = (char *)mbedtls_test_ca_crt_ec;
```

Comment:
mbedtls_test_ca_crt_ec is valid when MBEDTLS_ECDSA_C is defined. Is this always working when it's enabled?
If not, how about using `mbedtls_test_ca_crt`? Because it's always defined by certs.c line 408 and line 423.

Reply:
Done